### PR TITLE
2022 Accessibility chapter

### LINKED ISSUE #2889: Accessibility 2022
# Accessibility 2022

Accessibility illustration

If you're interested in contributing to the Accessibility chapter of the 2022 Web Almanac, please reply to this issue and indicate which role or roles best fit your interest and availability: [author](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide), [reviewer](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Reviewers'-Guide), [analyst](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide), and/or [editor](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide).

## Content team

| Lead | Authors | Reviewers | Analysts | Editors | Coordinator |
| ---- | ------- | --------- | -------- | ------- | ----------- |
|  | @SaptakS @scottdavis99 @thibaudcolas | @shantsis | @thibaudcolas | @dereknahman | @foxdavidj |

<details>
<summary><b>Expand for more information about each role 👀</b></summary>

- The **[content team lead](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Content-Team-Leads'-Guide)** is the chapter owner and responsible for setting the scope of the chapter and managing contributors' day-to-day progress.
- **[Authors](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide)** are subject matter experts and lead the content direction for each chapter. Chapters typically have one or two authors. Authors are responsible for planning the outline of the chapter, analyzing stats and trends, and writing the annual report.
- **[Reviewers](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Reviewers'-Guide)** are also subject matter experts and assist authors with technical reviews during the planning, analyzing, and writing phases.
- **[Analysts](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide)** are responsible for researching the stats and trends used throughout the Almanac. Analysts work closely with authors and reviewers during the planning phase to give direction on the types of stats that are possible from the dataset, and during the analyzing/writing phases to ensure that the stats are used correctly.
- **[Editors](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide)** are technical writers who have a penchant for both technical and non-technical content correctness. Editors have a mastery of the English language and work closely with authors to help wordsmith content and ensure that everything fits together as a cohesive unit.
- The **[section coordinator](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Section-Leads'-Guide)** is the overall owner for all chapters within a section like "User Experience" or "Page Content" and helps to keep each chapter on schedule.

_Note: The time commitment for each role varies by the chapter's scope and complexity as well as the number of contributors._

For an overview of how the roles work together at each phase of the project, see the [Chapter Lifecycle](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle) doc.
</details>

### [**0. Form the content team**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#0-create-content-team)
+ [x] **May 1**: The content team has at least one author, reviewer, and analyst

### [**1. Plan content**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#1-plan-content)
+ [x] **May 15** The content team has completed the chapter outline in the [draft doc][~google-doc]

### [**2. Gather data**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#2-gather-data)
+ [x] **June 1**: Analysts have added all necessary [custom metrics](https://github.com/HTTPArchive/custom-metrics/blob/main/README.md) and drafted a PR ([example](https://github.com/HTTPArchive/almanac.httparchive.org/pull/1087)) to track query progress
- **June 1 - 15**: HTTP Archive runs the June crawl

### [**3. Validate results**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#3-validate-results)
+ [x] **August 1**: Analysts have queried all metrics and saved the output to the [results sheet][~google-sheets]

### [**4. Draft content**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#4-draft-content)
+ [x] **September 1**: The content team has written, reviewed, and edited the chapter in the [doc][~google-doc]

### [**5. Publication**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#5-publication)
+ [x] **September 15**: The completed chapter and all required [metadata](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters) and [figures](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Figures-Guide) are converted to markdown and submitted to [GitHub][~chapter-markdown]
- **September 26**: Target launch date 🚀

## Chapter resources

Refer to these 2022 Accessibility resources throughout the content creation process:

[📄 Google Docs][~google-doc] for outlining and drafting content
[🔍 SQL files][~sql-files] for committing the queries used during analysis
[📊 Google Sheets][~google-sheets] for saving the results of queries
[📝 Markdown file][~chapter-markdown] for publishing content and managing public metadata
[💬 #web-almanac-accessibility on Slack][~slack-channel] for team coordination

[~google-doc]: https://docs.google.com/document/d/1Q6PEEfqD59vLgrOXY2VQUDeVrl0wcWHqsQiIhQ0Bt3M/edit?usp=sharing
[~sql-files]: https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2022/accessibility
[~google-sheets]: https://docs.google.com/spreadsheets/d/1ladaKh6RbtMKQwkccwxDJGQf85KyhfLrtlM_9e9sLH8/edit?usp=sharing
[~chapter-markdown]: https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/src/content/en/2022/accessibility.md
[~slack-channel]: https://join.slack.com/t/httparchive/shared_invite/zt-45sgwmnb-eDEatOhqssqNAKxxOSLAaA

### PR DESCRIPTION
**Pull request by tunetheweb**
Moved from #3085 to be part of this repo to make it easier for others to contribute to this branch.

Closes #2889 

Staged at: https://20220823t154507-dot-webalmanac.uk.r.appspot.com/en/2022/accessibility
Featured-Quote: https://20220823t154507-dot-webalmanac.uk.r.appspot.com//en/2022/?feat=accessibility#featured-chapter

### CODE REVIEW

**Comment by shantsis** [comment]
Are we making any suggestions/edits in GitHub or in the google doc? Just want to confirm

**Comment by tunetheweb** [comment]
Good question! Seems like the chapter isn't quite as complete as I presumed it was when the first draft PR was opened. Suggest you stick to the Doc for now, and can then copy into this branch when it's settled down more.

**Comment by shantsis** [comment]
Thanks! I’ll take a look tonight.

**Comment by shantsis** [comment]
I've left comments on the intro and ease of reading so far, will keep going tomorrow.

**Comment by thibaudcolas** [comment]
This is ready for review, with 3 things missing before it can be merged:

- Featured quote
- Featured stats
- There is a Lighthouse failure in CI I’m not understanding (on the 2022 "homepage"’s content width).

**Comment by rviscomi** [comment]
What's the status of this PR, is it ready to merge?

**Comment by tunetheweb** [comment]
Just did a first pass and it's looking pretty good. Will give it a proper read tomorrow and then merge.

File: src/content/en/2022/accessibility.md [hunk 1]
```diff
@@ -1,22 +1,791 @@
...
-#TODO - Review and update chapter description
-description: Accessibility chapter of the 2022 Web Almanac covering ease of reading, media, ease of navigation, and compatibility with assistive technologies.
-authors: [scottdavis99, SaptakS]
-reviewers: [shantsis, alextait1, ericwbailey]
-analysts: [scottdavis99, thibaudcolas]
+description: Accessibility chapter of the 2022 Web Almanac covering ease of reading, navigation, forms, media, ARIA, and accessibility apps.
+authors: [SaptakS, scottdavis99, thibaudcolas]
+reviewers: [shantsis]
+analysts: [thibaudcolas]
 editors: [dereknahman]
```
Comment (tunetheweb) [inline src/content/en/2022/accessibility.md:8]: @SaptakS can you confirm this list is correct and, within each category, contributors are listed in descending order of contribution?

**Review by tunetheweb — COMMENTED** [review]
This looks good to merge now!

File: src/content/en/2022/accessibility.md (hunk 1 above)
Comment (SaptakS) [reply src/content/en/2022/accessibility.md:8]: About the order, I would change authors order to [SaptakS, thibaudcolas, scottdavis99]. Rest looks good to me.

Comment (tunetheweb) [reply src/content/en/2022/accessibility.md:8]:
```suggestion
authors: [SaptakS, thibaudcolas, scottdavis99]
reviewers: [shantsis]
analysts: [thibaudcolas]
editors: [dereknahman]
```

**Pull request merged**